torch/_inductor/codegen/triton.py

```diff
@@ -1341,6 +1341,28 @@ def _new_line(self, line):
 
         return sympy_symbol(str(var))
 
+    def get_strides_of_load(self, index: sympy.Expr):
```

Comment:
any reason not to lift this to `common.py` and use it in triton and c++?

Reply:
Just don't have a particular need for it in C++ today.

Reply:
I meant
https://github.com/pytorch/pytorch/blob/f2d7faf4ba92a6ed43890775ca6ca174ddbf99ea/torch/_inductor/codegen/cpp.py#L293-L296